plugins/web-plugin/src/main/java/ru/sbtqa/tag/stepdefs/ru/WebSteps.java

```diff
@@ -85,6 +88,55 @@ public void reInitPage() {
         super.reInitPage();
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @And("^пользователь принимает уведомление с текстом \"(.*?)\"$")
+    public void acceptAlert(String text) throws WaitException {
+        super.acceptAlert(text);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @And("^пользователь отклоняет уведомление с текстом \"(.*?)\"$")
+    public void dismissAlert(String text) throws WaitException {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add the "@Override" annotation above this method signature ([squid:S1161](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1161))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=96&issues=AWZh47EwrFHbA8-Egpm8&open=AWZh47EwrFHbA8-Egpm8)

[ISSUE_KEY]: AWZh47EwrFHbA8-Egpm8